mpqp/core/instruction/barrier.py

```diff
@@ -31,3 +31,6 @@ def to_other_language(
             from qiskit.circuit.library import Barrier as QiskitBarrier
 
             return QiskitBarrier(self.size)
+        elif language == Language.QASM2:
+            qubits = ",".join([f"q[{j}]" for j in self.targets])
+            return "\nbarrier " + qubits + ";"
```

Comment:
u sure about the line break ? I would not put it here and instead add it at the circuit export level. Something like
```py
def to_other_language(self, language):
    if language == Language.QASM2:
        return "\n".join(self.instructions.to_other_language(Language.QASM2)
```